sv-comp/benchexec/my-bench/goblint.xml

```diff
@@ -9,6 +9,7 @@
   <option name="--disable">ana.int.def_exc</option>
   <option name="--enable">ana.int.enums</option>
   <option name="--enable">ana.int.interval</option>
+  <option name="--disable">ana.oct_no_uints</option>
```

Comment:
I wouldn't bother modifying this file, since it's not really used as we put all the SV-COMP configuration into `svcomp21.json` instead.